### PR TITLE
Always x synchronous 

### PR DESCRIPTION
**Pull request by lferran**
### Description

Not having synchronous writes is not an API we want anymore, as it is very easy to flood the ingestion system.
This PR makes all write operations synchronous.

### How was this PR tested?
Existing tests cover it
- [ ] add unit test on transaction class

### CODE REVIEW

File: nucliadb_utils/nucliadb_utils/transaction.py [hunk 1]
```diff
@@ -204,6 +208,7 @@ async def commit(
                 await asyncio.wait_for(waiting_event.wait(), timeout=30.0)
```
Comment (vangheem) [inline nucliadb_utils/nucliadb_utils/transaction.py:208]: what do you think about increasing this timeout?

**Review by vangheem — APPROVED** [review]

File: nucliadb/nucliadb/writer/api/v1/field.py [hunk 2]
```diff
@@ -108,19 +107,24 @@ async def finish_field_put(
...
-    set_processing_info(writer, processing_info)
-    await transaction.commit(writer, partition, wait=wait_on_commit)
-
-    return processing_info.seqid
+    try:
+        writer.source = BrokerMessage.MessageSource.WRITER
+        await transaction.commit(writer, partition, wait=True)
+    except TransactionCommitTimeoutError:
+        raise HTTPException(
+            status_code=501, detail="Inconsistent write. Commit timeout"
```
Comment (jotare) [inline nucliadb/nucliadb/writer/api/v1/field.py:119]: What about something more descriptive?

```suggestion
            status_code=501, detail="Inconsistent write. This resource will never be processed and may be never stored"
```

File: nucliadb/nucliadb/writer/api/v1/field.py [hunk 3]
```diff
@@ -721,17 +652,21 @@ async def _append_messages_to_conversation_field(
 
     await parse_conversation_field(field_id, field, writer, toprocess, kbid, rid)
 
+    writer.source = BrokerMessage.MessageSource.WRITER
+    try:
+        await transaction.commit(writer, partition, wait=True)
+    except TransactionCommitTimeoutError:
+        raise HTTPException(
+            status_code=501, detail="Inconsistent write. Commit timeout"
```
Comment (jotare) [inline nucliadb/nucliadb/writer/api/v1/field.py:660]: Same about more descriptive log

File: nucliadb/nucliadb/writer/api/v1/field.py [hunk 4]
```diff
@@ -818,17 +750,19 @@ async def _append_blocks_to_layout_field(
     field.body.blocks.update(blocks)
     await parse_layout_field(field_id, field, writer, toprocess, kbid, rid)
 
+    writer.source = BrokerMessage.MessageSource.WRITER
+    try:
+        await transaction.commit(writer, partition, wait=True)
+    except TransactionCommitTimeoutError:
+        raise HTTPException(
+            status_code=501, detail="Inconsistent write. Commit timeout"
```
Comment (jotare) [inline nucliadb/nucliadb/writer/api/v1/field.py:758]: Same

File: nucliadb/nucliadb/writer/api/v1/field.py [hunk 5]
```diff
@@ -908,7 +836,12 @@ async def _delete_resource_field(
     writer.delete_fields.append(pb_field_id)
     parse_audit(writer.audit, request)
 
-    await transaction.commit(writer, partition, wait=x_synchronous)
+    try:
+        await transaction.commit(writer, partition, wait=True)
+    except TransactionCommitTimeoutError:
+        raise HTTPException(
+            status_code=501, detail="Inconsistent write. Commit timeout"
```
Comment (jotare) [inline nucliadb/nucliadb/writer/api/v1/field.py:843]: Same (for this and all the other ones)

**Comment by lferran** [comment]
[sc-9547]

**Pull request merged**